cpp/tests/strings/combine_tests.cpp

```diff
@@ -281,8 +281,13 @@ TEST_F(StringsConcatenateWithColSeparatorTest, SingleColumnStringMixNoReplacemen
     {"", "", "", "bbabc", "", "d", "éa", "", "bbb", "éééf"},
     {false, false, true, true, false, true, true, false, true, true});
 
+  CUDA_TRY(cudaDeviceSynchronize());
```

Comment:
Why is this needed here?

Reply:
Good catch, must be leftover from debugging